styles.css

```diff
@@ -118,6 +118,4 @@
   article {
     width: 300px;
   }
```

Comment:
suggestion: ignore using direct tag names, just put class and give style to the class.